src/pkg/packaging-tools/sharedFramework/Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+  <!--
+    This Directory.Build.props prevents the SDK from skipping down to the root
+    Directory.Build.props, following the dir.props chain instead.
```

Comment:
Any reason not to just rename?

Reply:
Rename which thing?

Since the import is in the (new) SDK props, it seems most practical to me to keep this around until the entire project moves from `dir.props` to `Directory.Build.props` during the move to Arcade, then this particular one can be removed.

Reply:
If you mean rename the root `Directory.Build.props` so it isn't detected, I believe that one's actively used to build the managed packages. Might actually cause source-build problems too, come to think of it, since source-build has a `Directory.Build.props` and the SDK has no search boundaries.

Leaving it as-is for now, will get better with Arcade.